helm-chart/csi-driver/templates/linode-block-storage.yaml

```diff
@@ -7,5 +7,9 @@ metadata:
   annotations:
     storageclass.kubernetes.io/is-default-class: "true"
 {{- end }}
+{{- if .Values.addTags }}
+parameters:
+  linodebs.csi.linode.com/volumeTags: "{{ .Values.addTags }}"
```

Comment:
can this simply be "{{ .Values.volumeTags }}" instead of "addTags" , to match the parameter name?